openquake/baselib/tests/parallel_test.py

```diff
@@ -146,6 +147,10 @@ def tearDownClass(cls):
 
 
 class ThreadPoolTestCase(unittest.TestCase):
+    @unittest.skipIf(
+        sys.platform == 'darwin' and platform.processor() == 'i386',
+        reason=('Skipped on MacOS Intel (it would raise "socket.gaierror:'
+                ' [Errno 8] nodename nor servname provided, or not known"'))
```

Comment:
@micheles do you think it is acceptable to skip this?

Reply:
Yes, since the Intel macs are going to die.